src/main/java/de/nplay/moderationbot/permissions/CustomPermissionProvider.java

```diff
@@ -21,6 +22,10 @@ public boolean hasPermission(@NotNull User user, @NotNull Context context) {
 
     @Override
     public boolean hasPermission(@NotNull Member member, @NotNull Context context) {
+        if(member.hasPermission(Permission.ADMINISTRATOR)) {
```

Comment:
do we really want this? This might be a potential security risk

Reply:
You resolved this conversation? Did you changed your mind?

Reply:
yes changed my mind, I didn't find a realistic attack vector